package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shiftcommerce/shift-next",
-  "version": "0.10.0",
+  "version": "0.10.1",
```

Comment:
Just to let you know James has a PR waiting which bumps to version 0.11.0

Reply:
Cheers, conflicts will catch it. Ideally this one will get merged in first so that the current bug doesn't make it into other releases.